src/panels/lovelace/entity-rows/hui-input-number-entity-row.ts

```diff
@@ -74,43 +74,41 @@ class HuiInputNumberEntityRow extends LitElement implements LovelaceRow {
     }
 
     return html`
-      <hui-generic-entity-row .hass="${this.hass}" .config="${this._config}">
-        <div>
-          ${stateObj.attributes.mode === "slider"
-            ? html`
-                <div class="flex">
-                  <ha-slider
-                    .dir="${computeRTLDirection(this.hass!)}"
-                    .step="${Number(stateObj.attributes.step)}"
-                    .min="${Number(stateObj.attributes.min)}"
-                    .max="${Number(stateObj.attributes.max)}"
-                    .value="${Number(stateObj.state)}"
-                    pin
-                    @change="${this._selectedValueChanged}"
-                    ignore-bar-touch
-                    id="input"
-                  ></ha-slider>
-                  <span class="state">
-                    ${Number(stateObj.state)}
-                    ${stateObj.attributes.unit_of_measurement}
-                  </span>
-                </div>
-              `
-            : html`
-                <paper-input
-                  no-label-float
-                  auto-validate
-                  .pattern="[0-9]+([\\.][0-9]+)?"
+      <hui-generic-entity-row .hass=${this.hass} .config=${this._config}>
+        ${stateObj.attributes.mode === "slider"
+          ? html`
+              <div class="flex">
+                <ha-slider
```

Comment:
Huh? We have `ha-slider` and `ha-paper-slider` and they are both based on `paper-slider`...?

Reply:
I don't know 😄